dimagi/urls.py

```diff
@@ -61,11 +61,13 @@ def get_robots():
     url(r'^case-management/$', pages.case_management,  name='case_management'),
     url(r'^test_500/$', pages.test_500),
     url(r'^test_404/$', pages.test_404),
-    url(r'^referral-commcare/$', pages.referral_commcare, name='referral_commcare'),
+    # url(r'^referral-commcare/$', pages.referral_commcare, name='referral_commcare'), for future pull back
```

Comment:
it is always better to delete code rather than comment it out. it's easy to go back to an old commit and revert